src/pages/ValidateConfigFiles/ServiceTemplatesTab.js

```diff
@@ -307,6 +307,31 @@ class ServiceTemplatesTab extends Component {
     );
   }
 
+  async handleSesUpload(file) {
+    this.setState({ loading: true });
+    let fileContents = await readFile(file);
+    await postJson('/api/v2/service/files/ses/ses_config.yml', JSON.stringify(fileContents));
+    this.setState(prevState => ({
+      loading: false,
+      serviceFiles: prevState.serviceFiles.map(serviceFile => {
+        if(serviceFile.service === 'ses') {
+          serviceFile.files.push('ses_config.yml');
+          if(serviceFile.changedFiles) {
+            serviceFile.changedFiles.push('ses_config.yml');
+          } else {
+            serviceFile.changedFiles = ['ses_config.yml'];
+          }
+        }
+        return serviceFile;
+      })
+    }));
+    this.props.hasChange?.(true);
```

Comment:
If hasChange is a function call with those parameters.

Reply:
compiles to something like:
```javascript
this.props.hasChange === 'function' && this.props.hasChange(true);
```

Reply:
ok, that makes sense. It looked like function invocation, but I didn't realize hasChange was a function (it doesn't sound like one). Next time I'll download the whole PR so that I can see the context better